sw/host/opentitanlib/src/app/mod.rs

```diff
@@ -58,11 +60,11 @@ impl TransportWrapper {
         self.transport.borrow().gpio_pin(Self::map_name(&self.pin_map, name).as_str())
     }
 
-    /// Programs a bitstream into an FPGA.
-    pub fn fpga_program(&self, bitstream: &[u8]) -> Result<()> {
-        self.transport.borrow().fpga_program(bitstream)
+    /// Invoke non-standard functionality of some Transport implementations.
+    pub fn dispatch(&self, action: &mut dyn Any) -> Result<Option<Box<dyn Serialize>>> {
+        self.transport.borrow().dispatch(action)
```

Comment:
Why `mut`?

I think the `action` struct should almost always be plain-old-data.